tests/redirects/map_globalconf.py

```diff
@@ -280,9 +280,6 @@
     url_test('/thunderbird/about/{board,press,staff}/',
              'http://website-archive.mozilla.org/www.mozilla.org/thunderbird/thunderbird/about/{board,press,staff}/'),
 
-    # bug 1121082
-    url_test('/hello/', '/firefox/hello/'),
```

Comment:
There still might be people using older versions of Firefox. The number might be small, but sending them to the SUMO page to explain why Hello no longer works is probably better than a 404 page.